migration/orm.go

```diff
@@ -206,3 +206,23 @@ func migrate(
 	}
 	return nil
 }
+
+/*
+Migrate will query the current schema of the named package and attempt
+to Migrate the passed value up to the current value.
+
+Returns an error if the passed value is not Migratable,
+not registered with migrations, missing Metadata, has a Schema
+higher than currentSchema, if the final migrated value is invalid,
+or other such conditions.
+
+If this returns no error, you can safely use the contents of value in
+code working with the currentSchema.
+*/
```

Comment:
This is an awesome comment! :+1: 

[Convention](https://golang.org/doc/effective_go.html#commentary) is to use block comments mostly as block comments. I think this is better as a line comment :nail_care: